### PR TITLE
Setup linter

### LINKED ISSUE #10: Setup linter
Create Eslint and Prettier configuration files. Add Prettier and Eslint extensions to VSCode recommend extensions list.

### PR DESCRIPTION
**Pull request by tin0312**
fixes #10

### CODE REVIEW

File: .prettierrc.json
Comment (DukeManh) [inline .prettierrc.json:1]: Question: why is it blank?

Comment (tin0312) [reply .prettierrc.json:1]: Sorry I missed that, please check the updated PR

Comment (DukeManh) [reply .prettierrc.json:1]: Can you run the format command after this change.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "env": {
+    "browser": true,
```
Comment (DukeManh) [inline .eslintrc.json:3]: This should fix "chrome is not defined" error
```suggestion
    "browser": true,
    "webextensions": true,
```

**Review by DukeManh — APPROVED** [review]

**Comment by DukeManh** [comment]
Nice work, thank you.

**Pull request merged**